mobile/native/src/trade/mod.rs

```diff
@@ -1,3 +1,42 @@
+use bitcoin::Amount;
+use bitcoin::SignedAmount;
+use rust_decimal::Decimal;
+use time::OffsetDateTime;
+use trade::ContractSymbol;
+use trade::Direction;
+use uuid::Uuid;
+
 pub mod order;
 pub mod position;
 pub mod users;
+
+/// A trade is an event that moves funds between the Lightning wallet and a DLC channel.
+///
+/// Every trade is associated with a single market order, but an order can be associated with
+/// multiple trades.
+///
+/// If an order changes the direction of the underlying position, it must be split into _two_
```

Comment:
🤔 do we really have to reflect the technical details on the model? I wouldn't have bothered and simply acted as if it's one atomic action. Logically it doesn't make a difference does it?

Reply:
It was simpler just to avoid having to deal with trades that change the direction of the position. But we may want to revisit this decision.